src/res/translation/wininstaller/fr.nsi

```diff
@@ -0,0 +1,37 @@
+; French translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_ENGLISH} \
+    "Créer un raccourci sur le bureau"
+
+LangString INVALID_FOLDER_MSG ${LANG_ENGLISH} \
+    "Le dossier de destination existe déjà. Veuillez entrer un nouveau dossier de destination."
+
+LangString RUNNING_APP_MSG ${LANG_ENGLISH} \
+    "${APP_NAME} est en cours d'exécution. Veuillez le fermer et relancer l'installation."
+
+LangString OLD_WRONG_VER_FOUND ${LANG_ENGLISH} \
+    "Nous avons détecté une ancienne version de $ {APP_NAME} dans votre dossier Program Files 32 bits. Il est fortement recommandé de le supprimer avant d'installer une nouvelle version de $ {APP_NAME}. Voulez-vous le supprimer maintenant ? "
+
+LangString OLD_WRONG_VER_FOUND_CONFIRM ${LANG_ENGLISH} \
+    "Si vous continuez sans le supprimer, votre installation risque d'être interrompue ! Êtes-vous sûr de ne pas vouloir supprimer l'ancienne version ? "
+
+LangString OLD_VER_REMOVE_FAILED ${LANG_ENGLISH} \
+     "Désolé, nous n'avons pas pu désinstaller l'ancienne version. Nous essaierons d'installer la nouvelle version, mais vous pouvez également appuyer sur Annuler et essayer de supprimer l'ancienne version par vous-même."
+
+LangString ASIO_DRIVER_HEADER ${LANG_ENGLISH} \
+     "Pilote ASIO"
+
+LangString ASIO_DRIVER_SUB ${LANG_ENGLISH} \
+     "Pour utiliser Jamulus, vous avez besoin d'un pilote ASIO"
+
+LangString ASIO_DRIVER_EXPLAIN ${LANG_ENGLISH} \
+     "Jamulus a besoin d'un pilote ASIO pour fournir un son à faible latence. Plus d'informations : "
+
+LangString ASIO_DRIVER_MORE_INFO ${LANG_ENGLISH} \
+     "Plus d'informations à propos d'ASIO sur jamulus.io"
+
+LangString ASIO_DRIVER_MORE_INFO_URL ${LANG_ENGLISH} \
+     "https://jamulus.io/fr/wiki/Installation-for-Windows"
```

Comment:
Could you please also add the \#asio anchor here? This would then automatically jump to the correct place of the site once it’s translated.